src/pages/FriendNew.js

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from "react";
+import { useParams, useHistory } from "react-router";
+import { useSelector } from "react-redux";
+import styled from "styled-components";
+import theme from "../theme";
+
+import ModalWrapper from "../components/ModalWrapper";
+import DetailWrapper from "../components/DetailWrapper";
+
+import Button from "../components/Button";
+import CircleButton from "../components/CircleButton";
+import { sendById } from "../api/friend";
+
+const Wrapper = styled.div`
+  display: grid;
+  padding: 2rem;
+
+  .center {
+    text-align: center;
+  }
+
+  .board-layout {
+    display: flex;
+    flex-wrap: wrap;
+  }
+`;
+
+const BoardWrapper = styled.div`
+  display: flex;
+  width: 360px;
+  height: 260px;
+  margin: 15px 30px;
+  padding: 25px 20px 20px 20px;
+  border-radius: 5px;
+  box-shadow: 0 0 0 20px ${({ theme }) => theme.background.main };
+  background-color: ${(({ theme }) => theme.background.board )};
```

Comment:
불필요한 괄호가 하나 더 있는 것 같습니다. 
```suggestion
  background-color: ${({ theme }) => theme.background.board};
```